locale/de/settings.cfg

```diff
@@ -1,37 +1,37 @@
-ltn-dispatcher-delivery-timeout(s)=Lieferzeit in Sekunden bevor Züge als vermisst gelten.\ndefault = 600s (10min)
-ltn-dispatcher-requester-delivery-reset=Inaktiv: (default)\nLieferung und Fahrplan werden bei Ankunft im Depot zurück gesetzt.\nÄnderungen an Zügen in Anforderer-Haltestellen haben keine Auswirkung.\n\nAktiv:\nLieferung und Fahrplan werden bei verlassen der Anforderer-Haltestelle zurück gesetzt.\nÄnderungen an Zügen in Anforderer-Haltestellen löscht die Lieferung und setzt den Fahrplan zurück.
-ltn-dispatcher-finish-loading=Aktiv: (default)\nVerhindert verlassen von Haltestellen während Greifarme/Pumpen arbeiten.\n\nInaktiv:\nZüge verlassen Haltestellen unmittelbar nach erreichen ihrer gewünschten Beladung.\nGreifarme bleiben Gegenstände haltend stecken, Tankwaggons enthalten Restmengen.
-ltn-depot-reset-filters=Aktiv: (default)\nFilter und Stapelbegrenzung in Güterwagons werden im Depot gelöscht.
-ltn-stop-default-network=Netzwerk ID wenn kein Signal "Kodierte Netzwerk ID" existiert.
\ No newline at end of file
+ltn-dispatcher-requester-threshold="Mindestmenge an Artikeln um eine Lieferung auszulösen. \nKann durch Signal pro Anforderer-Haltestelle überschrieben werden.\nStandardwert = 1000"
+ltn-dispatcher-provider-threshold="Mindestmenge an Artikeln um als Anbieter erkannt zu werden. \nKann durch Signal pro Anbieter-Haltestelle überschrieben werden.\nStandardwert = 1000"
+ltn-dispatcher-schedule-circuit-control="Aktiviert Schaltungsbedinungen rot = 0 ODER grün ≥ 1 for alle Haltestellen.\nAchtung: Alle LTN Haltestellen müssen \"an Zug senden\" aktiviert und eine Schaltnetz-Verbindung haben. Sonst warten Züge ewig auf ein Signal.\nStandardwert = false"
+ltn-dispatcher-depot-inactivity(s)="Dauer in Sekunden, bevor Züge das Depot aufgrund von Inaktivität verlassen.\nStandardwert = 5"
+ltn-dispatcher-stop-timeout(s)="Dauer in Sekunden, bevor Züge ihre Station verlassen müssen.\n0 deaktiviert diese Funktion.\nStandardwert = 120"
+ltn-dispatcher-delivery-timeout(s)="Lieferzeit in Sekunden, bevor Züge als vermisst gelten.\nStandardwert = 600 (10 min)"
+ltn-dispatcher-requester-delivery-reset=Inaktiv: (Standardwert)\nLieferung und Fahrplan werden bei Ankunft im Depot zurück gesetzt.\nÄnderungen an Zügen in Anforderer-Haltestellen haben keine Auswirkung.\n\nAktiv:\nLieferung und Fahrplan werden bei verlassen der Anforderer-Haltestelle zurück gesetzt.\nÄnderungen an Zügen in Anforderer-Haltestellen löscht die Lieferung und setzt den Fahrplan zurück.
+ltn-dispatcher-finish-loading=Aktiv: (Standardwert)\nVerhindert verlassen von Haltestellen während Greifarme/Pumpen arbeiten.\n\nInaktiv:\nZüge verlassen Haltestellen unmittelbar nach erreichen ihrer gewünschten Beladung.\nGreifarme bleiben Gegenstände haltend stecken, Tankwaggons enthalten Restmengen.
+ltn-depot-reset-filters=Aktiv: (Standardwert)\nFilter und Stapelbegrenzungen in Güterwaggons werden beim Betreten eines Depots gelöscht.

```

Comment:
Ein Zug betritt kein Depot.

Reply:
Aber steht das nicht im englischen Text? "True: (default)\nCargo wagons have their filters and stack limitations cleared when entering a depot."
Oder verstehe ich dort was falsch?

Reply:
Die korrekte Übersetzung für "a train enters its depot" ist "ein Zug kommt in seinem Bahnbetriebswerk an"